src/it/java/com/zotov/edu/passportofficerestservice/model/PageResponse.java

```diff
@@ -0,0 +1,20 @@
+package com.zotov.edu.passportofficerestservice.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@JsonIgnoreProperties
+public class PageResponse<T> {
```

Comment:
Let's try to implement custom serializer for Spring's `Page` instead of custom DTO

Reply:
Implemented a custom Page serializer